### DISCUSSION TITLE
Renderer compatibility issues

### DISCUSSION THREAD

**Discussion by Horbin-Magician** [slint-ui/slint#5799] [Q&A]
I used different backends, including: winit-femtovg, renderer-skia, renderer-skia-opengl.
My program runs on different graphics environments (all in windows), including: AMD core graphics card, Intel core graphics card, NVIDIA card.

All renderers have some problems:
1. winit-femtovg: **unclear font** when use core graphics card (both AMD and Intel);
2. winit-femtovg: The word input field will **block lineedit** when using pinyin;
3. winit-femtovg and renderer-skia-opengl: **transparent window failure** when use part AMD core graphics card;
4. renderer-skia-opengl: **Path is out of work** when use AMD core graphics card;
5. renderer-skia: **transparent window failure** when use AMD core graphics card;
(These are some of the problems I've found these days)

I've tried "This application only uses high performance graphics cards" feature, but these problems still occur.
Only when I completely blocked out the core graphics card that all functions were mostly back to normal.
It seems that all of these renderers are currently only well supported on nvidia cards.
What can I do to improve their performance on the core graphics card?

**Comment by tronical** [comment]
I'm afraid that I can't offer a "silver bullet" (I wonder if there's a better phrase...).

If translucency doesn't work, then I'd say that's a bug. I believe a ticket exists for translucency with the Direct3D backend? Similarly, path rendering not working with Skia's OpenGL renderer sounds like a bug (I haven't heard of that one yet though).

I don't mind having an open ended discussing here, maybe others can chime in with experience and expertise. Meanwhile, help is definitely welcome to triage such issues and file them accordingly. (disclaimer: I don't have access to an AMD card, although either way we merely delegate to Skia)
